server/dash/dashboards/nc_summary_comparison.py

```diff
@@ -0,0 +1,842 @@
+    """
+    histogram = html.Div(
+        # Histogram for the request timeToClose.
+        dcc.Graph(
+            id="time_close_histogram", style={"height": "40vh",
+                     "width": "48vw"}
+        ), style=merge_dict(CHART_OUTLINE_STYLE, {"width": "48vw",
+                 "height": "40vh"}))
+    return histogram
+
+
+def generate_council_name_dropdown():
+    """Generates the neighborhood council (nc) dropdown for the 
+    comparison dashboard.
+
+    This functions generates the html elements for the 
+    nc dropdown for the left side of the comparison dashboard.
+
+    Return:
+        Dash html div element containing nc drop down for left pane filtering.
+    """
+    dropdown = html.Div(dcc.Dropdown(sorted([n for n in set(api_data_df["councilName"])]),
+         " ", id="nc_comp_dropdown",
+                 placeholder="Select a Neighborhood Council..."),
+                 style=merge_dict(INLINE_STYLE, {"width": "48.5vw"})),
+    return dropdown
+
+
+def generate_council_name_dropdown2():
+    """Generates the neighborhood council (nc) dropdown for the 
+    comparison dashboard.
+
+    This functions generates the html elements for the 
+    nc dropdown for the right side of the comparison dashboard.
+
+    Return:
+        Dash html div element containing nc drop down for right pane filtering.
+    """
+    dropdown = html.Div(dcc.Dropdown(sorted([n for n in set(api_data_df["councilName"])]),
+         " ", id="nc_comp_dropdown2",
+                 placeholder="Select a Neighborhood Council..."),
+                 style=merge_dict(INLINE_STYLE, {"width": "48.5vw"}))
+    return dropdown
+
+
+def generate_comparison_total_req():
+    """Generates the indicator visual for the 
+    total number of requests.
+
+    This functions generates the html elements for the 
+    total number of requests for the left side of the 
+    comparison dashboard.
+
+    Return:
+        Dash html div element containing label and indicator visual.
+    """
+    total_req = html.Div([
+        html.H6("Total Number of Requests", style=CENTER_ALIGN_STYLE),
+        html.H1(id="total_req_card", style=CENTER_ALIGN_STYLE)],
+        style=merge_dict(CHART_OUTLINE_STYLE, {"width": "24vw", "height": "16vh"}))
+    return total_req
+
+
+def generate_comparison_num_days():
+    """Generates the indicator visual for the 
+    total number of days request spans.
+
+    This functions generates the html elements for the 
+    total number of days request spans for the left side of the 
+    comparison dashboard.
+
+    Return:
+        Dash html div element containing label and indicator visual.
+    """
+    num_days = html.Div([
+        html.H6("Number of Days", style=CENTER_ALIGN_STYLE),
+        html.H1(id="num_days_card", style=CENTER_ALIGN_STYLE)],
+        style=merge_dict(CHART_OUTLINE_STYLE, {"width": "24vw", "height": "16vh"}))
+    return num_days
+
+
+def generate_comparison_total_req2():
```

Comment:
same comment about adding an arg instead of making a copy

Reply:
Thanks for the suggestion - resolved